src/app.js

```diff
@@ -1,16 +1,29 @@
 import express, { json, urlencoded } from 'express';
 import cors from 'cors'
 import morgan from 'morgan'
+import mongoose from 'mongoose'
+import DB from './config';
+//import route from './routes/index';
 
 const app = express();
+mongoose.connect(DB)
+    .then(() => {
+        console.log('Successful connected to MongoDB ATLAS');
+    })
+    .catch((error) => {
+        console.log("Unable to connect to MongoDB ATLAS");
+        console.error(error);
+   })
```

Comment:
abstract this to a file of its own and then call it in here, you can ignore that file in your tests to make it pass